django/contrib/admin/views/autocomplete.py

```diff
@@ -24,11 +25,18 @@ def get(self, request, *args, **kwargs):
             return JsonResponse({'error': '403 Forbidden'}, status=403)
 
         self.term = request.GET.get('term', '')
+        self.to_field = request.GET.get('to_field', None)
```

Comment:
```suggestion
        to_field = request.GET.get('to_field', None)
```

There is no need to set an attribute on the instance, since you are not using the value outside this function.